### PR TITLE
Wazo 1876 crud sip template

### PR DESCRIPTION
**Pull request by fblackburn1**

### CODE REVIEW

File: wazo_ui/plugins/conference/view.py [hunk 1]
```diff
@@ -17,7 +17,6 @@
 
 class ConferenceView(BaseIPBXHelperView):
 
-    resource_name = 'conference'
```
Comment (pc-m) [inline wazo_ui/plugins/conference/view.py:20]: Did you mean to do this change? I don't care if it's unrelated. I just want to make sure it won't break something.

File: wazo_ui/plugins/sip_template/form.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    label = StringField(l_('Label'), [Length(max=128)])
+    aor_section_options = FieldList(FormField(BasePJSIPOptionsForm))
+    auth_section_options = FieldList(FormField(BasePJSIPOptionsForm))
+    endpoint_section_options = FieldList(FormField(BasePJSIPOptionsForm))
+    identify_section_options = FieldList(FormField(IdentifyPJSIPOptionsForm))
+    registration_section_options = FieldList(FormField(RegistrationPJSIPOptionsForm))
+    registration_outbound_auth_section_options = FieldList(FormField(BasePJSIPOptionsForm))
+    outbound_auth_section_options = FieldList(FormField(BasePJSIPOptionsForm))
+    transport = FormField(TransportForm)
+    template_uuids = SelectMultipleField(l_('Templates'), choices=[])
```
Comment (pc-m) [inline wazo_ui/plugins/sip_template/form.py:70]: I would change the label to `Parent Templates`. `Templates` only is not so obvious when editing a template

File: wazo_ui/plugins/conference/view.py (hunk 1 above)
Comment (fblackburn1) [reply wazo_ui/plugins/conference/view.py:20]: Yes [see this commit](https://github.com/wazo-platform/wazo-ui/pull/66/commits/5c2e99b7725db67bfdf43dd15a0f3b17736340e4)

File: wazo_ui/plugins/sip_template/form.py (hunk 2 above)
Comment (fblackburn1) [reply wazo_ui/plugins/sip_template/form.py:70]: fixed

**Comment by fblackburn1** [comment]
> When choosing a configuration option the filter does not work.

Yes it is the same behavior for transport sip dropdown. It's outscope of this ticket and should be fixed independently

**Pull request merged**